data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -1,5 +1,433 @@
+        # If more levels should be supported, a loop can be added, but that will make the code more complex.
+        [store_unit]
+            [filter]
+                ability=tailwind
+                [filter_side]
+                    [allied_with]
+                        side=$side_number
+                    [/allied_with]
+                [/filter_side]
+                [and]
+                    # filter_wml is slow, which is why there's already an ability=tailwind filter before this
+                    [filter_wml]
+                        [abilities]
+                            [dummy]
+                                id=tailwind
+                                value=2
+                            [/dummy]
+                        [/abilities]
+                    [/filter_wml]
+                [/and]
+            [/filter]
+            variable=utbs_tailwind2_sources
+        [/store_unit]
+        [store_unit]
+            [filter]
+                ability=tailwind
+                [filter_side]
+                    [allied_with]
+                        side=$side_number
+                    [/allied_with]
+                [/filter_side]
+                [and]
+                    # filter_wml is slow, which is why there's already an ability=tailwind filter before this
+                    [filter_wml]
+                        [abilities]
+                            [dummy]
+                                id=tailwind
+                                value=1
+                            [/dummy]
+                        [/abilities]
+                    [/filter_wml]
+                [/and]
+            [/filter]
+            variable=utbs_tailwind1_sources
+        [/store_unit]
+
+        [modify_unit]
+            [filter]
+                side=$side_number
+                [filter_adjacent]
+                    find_in=utbs_tailwind2_sources
+                [/filter_adjacent]
+            [/filter]
+            moves="$($this_unit.moves + 2)"
+        [/modify_unit]
+        [modify_unit]
+            [filter]
+                side=$side_number
+                [filter_adjacent]
+                    find_in=utbs_tailwind1_sources
+                [/filter_adjacent]
+                [not]
+                    [filter_adjacent]
+                        find_in=utbs_tailwind2_sources
+                    [/filter_adjacent]
+                [/not]
+            [/filter]
+            moves="$($this_unit.moves + 1)"
+        [/modify_unit]
+
+        {CLEAR_VARIABLE utbs_tailwind1_sources,utbs_tailwind2_sources}
+    [/event]
+#enddef
+
+#define STR_TAILWIND_RESTRICT_1
+_ "At the start of the turn this unit increases movement points of surrounding units by +1"#enddef
+
+#define STR_TAILWIND_RESTRICT_2
+_ "At the start of the turn this unit increases movement points of surrounding units by +2"#enddef
+
+#define UTBS_TAILWIND_DESCRIPTION X
```

Comment:
UTBS_ABILITY_TAILWIND is now so small that the description could go in it, I don't see why this needs a separate macro.

However, the ability probably still needs to be changed from `[dummy]` to `[tailwind]`.

Reply:
How can I do that?

Reply:
It's literally what he said. Change `[dummy]` to `[tailwind]` and that's it.

Although I guess it also needs to be added to the schema as an ability that takes a value.

Reply:
Done, also the macro was created to make translation easier, requested by CelticMinstrel in one of the earlier reviews, I dont know if it is still necessary but I still kept it in the same way just to be safe.